task/buildah-rhtap/0.1/buildah-rhtap.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   annotations:
     tekton.dev/pipelines.minVersion: "0.12.1"
     tekton.dev/tags: "containers, rhtap"
+    build.appstudio.redhat.com/expires-on: 2024-11-13T00:00:00Z
```

Comment:
buildah-rhtap also has no 0.2 version and we can't deprecate it

I have to finish https://github.com/konflux-ci/build-definitions/pull/1195 first :smile:

Reply:
Okay, I thought it'd be fine since it would get merged before that. But you're right that it makes more sense to deprecate only **after** there's something to migrate to.